compiler/src/iree/compiler/Codegen/Common/CPU/CPULowerToUKernels.cpp

```diff
@@ -392,7 +392,7 @@ matchDAGForUKernel(RewriterBase &rewriter, tensor::PackOp op,
       ValueRange{in_size0, in_size1, out_size0, out_size1, out_size2, out_size3,
                  paddingVal, flagsVal},
       /*fn_def_attrs=*/rewriter.getDictionaryAttr(fn.defAttrs),
-      /*strided_outer_dims=*/rewriter.getIndexAttr(1));
+      /*strided_outer_dims=*/rewriter.getIndexAttr(2));
```

Comment:
Why is it `2`? Do we need a similar change in mmt4d ukernel conversion?

Reply:
This `strided_outer_dims` attribute is a single integer govering all strides for all operands of the ukernel. Since we want 2 strides for the input, we have to set it to 2, and then we will get an extra stride passed for the output too, but that's OK, we will simply not use it. The pack output is still not strided beyond the outermost dimension.